TelepresenceBot/core/src/main/java/com/novoda/tpbot/support/Observable.java

```diff
@@ -33,22 +36,26 @@ protected synchronized void notifyOf(T newValue) {
         }
     }
 
-    protected synchronized void detachObservers() {
+    public synchronized void detachObservers() {
         observers.clear();
     }
 
-    protected synchronized void setChanged() {
-        hasChanged = true;
-    }
+    public abstract Observable<T> start();
```

Comment:
I agree, being able to see instantly see what you need to implement is a good idea :+1: